content/es/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -0,0 +1,748 @@
+---
+title: Ciclo de vida de un Pod
+content_type: concept
+weight: 30
+---
+
+<!-- overview -->
+Esta página describe el ciclo de vida de un Pod.
+Los Pods siguen un ciclo de vida definido, comenzando en la fase [`Pending`](
+#pod-phase),
+y luego pasando a "en ejecución" `Running` si al menos uno de sus contenedores primarios se
+inicia correctamente,
+y luego pasando a "exitoso"(`Succeeded`) o "fallido" (`Failed`) si uno de los contenedores de un Pod
+termina en error.
+
+Mientras un Pod está en `Running`,
+el kubelet puede reiniciar sus contenedores para manejar algunos errores.
+Dentro de un Pod,
+Kubernetes rastrea diferentes [estados](#container-states) de contenedores y
+decide qué acción tomar para que el Pod esté sano otra vez.
+
+En la API de Kubernetes, los Pods tienen una especificación y un estado actual.
+El estado de un Pod consiste en un conjunto
+de [condiciones de un Pod](#pod-conditions).
+También puedes
+inyectar [información de estado personalizada](#pod-readiness-gate) en los datos
+de condiciones de un Pod, si es útil para tu aplicación.
+
+Los Pods se [programan](/docs/concepts/scheduling-eviction/) únicamente una vez
+en su tiempo de vida.
+Una vez que un Pod se programa (asigna) a un Nodo, el Pod se ejecuta en ese Nodo
+hasta que se termine o se [elimina](#pod-termination).
+
+<!-- body -->
+
+## Ciclo de vida de un Pod
+
+Igual que contenedores de aplicación individuales,
+se considera que los Pods son entidades relativamente efímeras
+(en lugar de durables).
+Los Pods se crean y se les 
+asigna un identificador único
+([UID](/docs/concepts/overview/working-with-objects/names/#uids)),
+y se programan para ejecutarse en nodos donde se mantienen hasta que se terminan
+(de acuerdo con las políticas de reinicio) o se eliminan.
+
+Si un {{< glossary_tooltip term_id="node" text="nodo" >}} muere,
+los Pods programados para ejecutarse en ese Nodo
+se [programan para eliminarse](#pod-garbage-collection) luego de un periodo de
+tiempo.
+
+Los Pods, por sí mismos, no se curan automáticamente.
+Si un Pod está programado para un {{< glossary_tooltip text="node"
+term_id="node" >}} y luego falla,
```

Comment:
Please avoid splitting the Hugo [shortcode](https://gohugo.io/content-management/shortcodes/) into multiple lines.

```suggestion
Si un Pod está programado para un {{< glossary_tooltip text="node" term_id="node" >}} y luego falla,
```